### PR TITLE
[Assembly] homepage refresh

### PR DESCRIPTION
**Pull request by alexcarpenter**
[🎟 Asana](https://app.asana.com/0/1200641486648086/1200899263951566)
[🎨 Figma](https://www.figma.com/file/PqaplzlVmvTFV9S5UDDSSE/Boundary-.io?node-id=1741%3A535)
[🔎 Preview](https://boundary-1ebu799ej-hashicorp.vercel.app/)

### CODE REVIEW

File: website/pages/home/index.jsx [hunk 1]
```diff
@@ -112,78 +125,102 @@ export default function HomePage() {
...
-                image: {
-                  url: require('./img/red-usecase-accessprivileges.png?url'),
-                },
-                link: {
-                  title: 'Learn more',
-                  url:
-                    'https://learn.hashicorp.com/tutorials/boundary/manage-users-groups',
-                },
+                alt: 'SSH',
+                url: require('./img/logos/ssh.png?url'),
```
Comment (BrandonRomano) [inline website/pages/home/index.jsx:154]: Could we export these out at SVGs?

File: website/pages/home/index.jsx [hunk 2]
```diff
@@ -112,78 +125,102 @@ export default function HomePage() {
...
+        <HowBoundaryWorks
+          heading="How Boundary Works"
+          description="HashiCorp Boundary is a secure remote access solution that provides an easy way to allow access to applications and critical systems with fine-grained authorizations based on trusted identities. Across clouds, local data centers, low-trust networks, Boundary provides an easier way to protect and safeguard access to application and critical systems by trusted identities without exposing the underlying network"
+          items={[
+            'Platform-agnostic proxy for dynamic targets',
+            'No SSH keys or VPN credentials to manage',
+            'Just-in-time credentials via HashiCorp Vault',
+          ]}
+          img={{
+            src: require('./img/how-boundary-works.png'),
```
Comment (BrandonRomano) [inline website/pages/home/index.jsx:219]: This one too is a little blurry; curious if we can export as an SVG here?

**Review by BrandonRomano — APPROVED** [review]
Looks good... Curious if we could change some of those PNGs to SVGs..

Beyond that, has my approval!

File: website/pages/home/index.jsx (hunk 2 above)
Comment (alexcarpenter) [reply website/pages/home/index.jsx:219]: This is actually a screenshot in the design file, so I wasn't able to convert to SVG.

File: website/pages/home/index.jsx (hunk 1 above)
Comment (alexcarpenter) [reply website/pages/home/index.jsx]: Got one https://github.com/hashicorp/boundary/pull/1558/commits/5280899f69e62a077b8734a3d8fa023c88adc50f

File: website/pages/home/index.jsx (hunk 2 above)
Comment (alexcarpenter) [reply website/pages/home/index.jsx]: Got it https://github.com/hashicorp/boundary/pull/1558/commits/f7dbdb60a18a34b92761562eec33b37729644da2

**Review by BrandonRomano — APPROVED** [review]

**Pull request merged**